.github/copilot-instructions.md

```diff
@@ -0,0 +1,235 @@
+# Plugin.Maui.DebugOverlay
+
+Plugin.Maui.DebugOverlay is a .NET MAUI plugin that provides debug overlay functionality including a visual debug ribbon and comprehensive visual tree dumping capabilities for MAUI applications.
+
+Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.
+
+## Working Effectively
+
+### Repository Setup and Requirements
+- Install .NET 8.0 or later SDK
+- Install .NET MAUI workload: `dotnet workload install maui`
+- CRITICAL: Repository currently targets .NET 10 (not yet released) causing build issues
+- Repository has missing MauiVersion property definition in build configuration
+
+### Build Process - CURRENT LIMITATIONS
```

Comment:
you should be able to build from linux for net9.0-android in CI. None of the other platforms probably will work...if you want to setup CI